src/hotspot/os/aix/attachListener_aix.cpp

```diff
@@ -285,8 +285,8 @@ AixAttachOperation* AixAttachListener::read_request(int s) {
   // Read until all (expected) strings have been read, the buffer is
   // full, or EOF.
 
-  int off = 0;
-  int left = max_len;
+  ssize_t off = 0;
+  ssize_t left = max_len;
```

Comment:
Doesn't this cause a sign-conversion warning because read expects size_t?

Reply:
I don't have -Wsign-conversion on, only -Wconversion, so I don't get -Wsign-conversion warnings for hotspot files.  This same change is made to os/linux/attachListener_linux.cpp.  Suggestions how to fix?

Reply:
checked_cast<size_t>() where necessary, unfortunately.